app/screens/ChildSetupList.tsx

```diff
@@ -64,14 +69,20 @@ const styles = StyleSheet.create({
   touchableRight: { 
     marginRight: 2,
     padding: 8 
+  },
+  listBgColor:{
+    backgroundColor:'red'
   }
 })
 const ChildSetupList = ({ navigation }: Props): any => {
   const netInfo = useNetInfoHook();
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
+  const [parentViewHeight, setParentViewheight] = useState(0);
+  const [profileViewHeight, setProfileViewheight] = useState(0);

```

Comment:
Let's follow camel case naming convention: 
setParentViewHeight
and
setProfileViewHeight